lib/ansible/modules/cloud/amazon/lambda.py

```diff
@@ -227,6 +247,47 @@ def sha256sum(filename):
 
     return hex_digest
 
+def set_tag(client, module, tags, function):
```

Comment:
There is a new compare_aws_tags function in module_utils.ec2 that should reduce the amount of work needed here